test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -35,85 +40,78 @@ describe('33acrossBidAdapter:', function () {
   });
 
   describe('isBidRequestValid:', function () {
-    context('valid bid request:', function () {
```

Comment:
Had to remove wrapping specs in a context since doing so failed nesting rules in linter

Reply:
Interesting.  Something along the lines of "there aren't enough test cases in this `context` / `describe`", or something like that?

Reply:
Oh no, 'twas too many nested calls. Their rules allowed a max of 3